src/test/java/org/entando/kubernetes/controller/coordinator/inprocesstests/EntandoOperatorMatcherTest.java

```diff
@@ -0,0 +1,103 @@
+/*
+ *
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ *  This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ *
+ */
+
+package org.entando.kubernetes.controller.coordinator.inprocesstests;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collections;
+import java.util.HashMap;
+import org.entando.kubernetes.controller.EntandoOperatorConfigProperty;
+import org.entando.kubernetes.controller.coordinator.EntandoOperatorMatcher;
+import org.entando.kubernetes.model.app.EntandoApp;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Tags;
+import org.junit.jupiter.api.Test;
+
+@Tags({@Tag("in-process"), @Tag("unit")})
+public class EntandoOperatorMatcherTest {
+
+    @BeforeEach
+    @AfterEach
```

Comment:
I'm not sure if it is an oversight or not. There is a reason to execute these lines before and after?

Reply:
Indeed this was intentional. Each test case assumes that the relevant system properties are unset, hence the BeforeEach. But at the same time, we can't leave the system properties there after the test cases, as it will affect the behaviour of the rest of our tests. Hence the AfterEach